torch/onnx/symbolic_opset9.py

```diff
@@ -523,16 +526,21 @@ def prim_ConstantSplit(g, self, split_size, dim):
 # TODO: Once we have proper scoping, stop reimplementing chunk, delete this
 # method, and use the desugared version
 def prim_ConstantChunk(g, self, chunks, dim):
-    split_size = (self.type().sizes()[dim] + chunks - 1) // chunks
+    dim_size = sym_help._get_tensor_dim_size(self, dim)
+    if dim_size is None:
+        return _unimplemented('prim::ConstantChunk', 'unknown dimension size')
+    split_size = (dim_size + chunks - 1) // chunks
     return prim_ConstantSplit(g, self, split_size, dim)
 
 
 @parse_args('v', 'i', 'i', 'i')
 def unsafe_chunk(g, self, chunks, dim, _outputs=None):
     if _outputs is None:
         return sym_help._onnx_opset_unsupported_detailed('unsafe_chunk', 9, 11, 'Dynamic number of outputs not supported')
-    split_size = (self.type().sizes()[dim] + chunks - 1) // chunks
-    size = self.type().sizes()[dim]
+    size = sym_help._get_tensor_dim_size(self, dim)
```

Comment:
Do we have a unit test for chunk?

Reply:
no, there is no explicit way of creating unsafe_unchunk. this is internally used by LSTM/GRU/RNN.